model/model-verifier/src/main/java/io/spine/model/verify/ModelVerifier.java

```diff
@@ -98,19 +101,19 @@ void verify(CommandHandlers spineModel) {
             "CheckReturnValue" /* Returned values for asXxxClass() are ignored because we use
                                   these methods only for verification of the classes. */
     })
-    private void verifyClass(Class<?> cls) {
+    private static void verifyClass(Class<?> cls) {
         Logger log = log();
         if (Aggregate.class.isAssignableFrom(cls)) {
             Class<? extends Aggregate> aggregateClass = (Class<? extends Aggregate>) cls;
-            model.asAggregateClass(aggregateClass);
+            asAggregateClass(aggregateClass);
```

Comment:
Instead of `static` import with this method name I'd rather have `AggregateClass.of(aggregateClass)` and use no static import.